language/tests/regent/run_pass/paral_saveobj.rg

```diff
@@ -0,0 +1,40 @@
+-- Copyright 2018 Stanford University
```

Comment:
Please name this file in complete words e.g. `compile_parallel_saveobj.rg`. Same for the other test.

Reply:
Done